resources/server/db/db_wrapper.ts

```diff
@@ -38,8 +44,8 @@ class _DbInterface {
    * @param values Variable definition
    **/
   public async exec(query: string, values?: unknown) {
-    const [res] = await this._internalQuery(query, values);
-    return (<ResultSetHeader>res).affectedRows;
+    const [res] = (await this._internalQuery(query, values)) as ResultSetHeader[];
+    return res.affectedRows;
   }
```

Comment:
The exec methods return types here could probably be done a little bit better here, using passed generics or something but tbh this works too, just explicit type assertion can feel a little *hacky* when compared to a generic system imo, need to think on long term implementation details but as a little hack this is fine.